crates/unleash-edge-metrics/src/send_unleash_metrics.rs

```diff
@@ -0,0 +1,192 @@
+use std::sync::Arc;
```

Comment:
This file is lifted directly from main, the only change here is that we no longer send to the legacy endpoint